src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/api/model/SupplementaryRiskDto.kt

```diff
@@ -7,22 +7,22 @@ import java.util.UUID
 data class SupplementaryRiskDto(
 
   @Schema(description = "Supplementary Risk ID", example = "78beac68-884c-4784-9bea-fd8088f52a47")
-  val supplementaryRiskId: UUID = UUID.randomUUID(),
+  val supplementaryRiskId: UUID? = null,
```

Comment:
why could this be nullable?

Reply:
because when the client POSTs they shouldn't send us this ID. It's our ID to create